cpp/lib/src/DbConnection.cc

```diff
@@ -463,17 +461,17 @@ void DbConnection::init(const std::string &database_name, const std::string &use
     type_ = T_MYSQL;
 
     if (::mysql_init(&mysql_) == nullptr)
-        throw std::runtime_error("in DbConnection::init: mysql_init() failed!");
+        LOG_ERROR("mysql_init() failed! (1)");
 
     if (::mysql_real_connect(&mysql_, host.c_str(), user.c_str(), passwd.c_str(), database_name.c_str(), port,
                              /* unix_socket = */nullptr, /* client_flag = */CLIENT_MULTI_STATEMENTS) == nullptr)
 
         // password is intentionally omitted here!
-        throw std::runtime_error("in DbConnection::init: mysql_real_connect() failed! (" + getLastErrorMessage()
-                                 + ", host=\"" + host + "\", user=\"" + user + "\", passwd=\"********\", database_name=\""
-                                 + database_name + "\", port=" + std::to_string(port) + ")");
+        LOG_ERROR("mysql_real_connect() failed! (" + getLastErrorMessage()
```

Comment:
The password has been intentionally omitted from log file due to security reasons, see comment on line 469